src/Files.App/Views/Pages/PropertiesGeneral.xaml

```diff
@@ -360,7 +360,7 @@
 					Width="16"
 					Height="16"
 					Margin="0,0,12,0"
-					Fill="{ThemeResource SystemAccentColorLight2}"
+					Fill="{Binding Foreground, ElementName=SpaceUserProgressRing}"
```

Comment:
What was wrong with the previous fill?

Reply:
In light mode, the color of the ring and its legend do not match for the space used. The SystemAccentColorLight2 color is used for dark mode but not for light mode. This change ensures that the legend always matches the ring.

Before
![Before_Property_Light](https://user-images.githubusercontent.com/46631671/197378750-606e85d2-ba41-4329-9753-b98502af6a61.png)
After
![After_Property_Light](https://user-images.githubusercontent.com/46631671/197378748-51065c05-ea58-4306-b058-bb15e1373d91.png)